.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,34 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          container-runtime: remote
+          runtime-cgroups: /system.slice/crio.service
       ---
-      kind: KubeletConfiguration
       apiVersion: kubelet.config.k8s.io/v1beta1
-      cgroupDriver: cgroupfs
+      kind: KubeletConfiguration
+      cgroupDriver: systemd
+      cgroupRoot: /
+      systemCgroups: /system.slice
+      kubeletCgroups: /system.slice/kubelet.service
+      enforceNodeAllocatable:
+      - pods
+      #- kube-reserved # try again when we bump system to get cgroupsv2
+      #- system-reserved # try again when we bump system to get cgroupsv2
+      systemReservedCgroup: /system.slice
+      kubeReservedCgroup: /system.slice/kubelet.service
+      systemReserved:
+      #  cpu: "200m" # try again when we bump system to get cgroupsv2
+        ephemeral-storage: 1Gi
+      #kubeReserved:
+      #  cpu: "200m" # try again when we bump system to get cgroupsv2
+      #  ephemeral-storage: "1Gi"
+      serializeImagePulls: false
+      containerLogMaxSize: 50Mi
+      #cpuManagerPolicy: static # try again when we bump system to get cgroupsv2
+      maxPods: 50
```

Comment:
Are these just experimental values? Or is there some specific reasoning behind them? If so, could you add it to the PR description?

Reply:
a baseline to have a starting point